src/NextLink.js

```diff
@@ -0,0 +1,42 @@
+import React, { forwardRef } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Link from 'next/link';
+import './NextLink.css';
```

Comment:
When we run our build script, css files won't be added to the 'es' and 'lib' folders.

One solution, and from what we have seen might be the "best" one, is to have a `dist` folder where all css would be placed and then each developer would import it the needed css files in way that best suits their project needs.

Reply:
yeah, I vouch for that as well and it reinforces the fact that the styles should not be mandatory.

Reply:
Done 👍